check-code/Makefile

```diff
@@ -0,0 +1,5 @@
+test: .cabal-sandbox
```

Comment:
That doesn't work because of this: https://github.com/zalora/tinc/issues/39.

I'll remove the `Makefile` and put a call to `tinc` into the bash script.